src/CSLOLUtils.cpp

```diff
@@ -119,8 +119,79 @@ bool CSLOLUtils::isUnnecessaryAdmin() {
     return result;
 }
 #else
```

Comment:
There is going to have to be ``#elif defined(__APPLE__)`` here now since this is macos specific implementation.

Reply:
I just cloned the repository and didn't change that myself, but I'll add that in.